config/global.js

```diff
@@ -42,6 +42,7 @@ const {
 	SC_DEMO_USER_NAME = 'schueler@schul-cloud.org',
 	SC_SUPERHERO_USER_PASSWORD = 'Schulcloud1!',
 	SC_SUPERHERO_USER_NAME = 'superhero@schul-cloud.org',
+	FEATURE_STUDENTS_CANT_BE_TEAM_ADMINISTRATOR_ENABLED,
```

Comment:
default.schema.json it the newer way to add envs and you must not add this in this file too.